### ISSUE TITLE
Vibration notification settings do not get saved

### ISSUE THREAD

**Issue by svelle** [mattermost/mattermost-mobile#1688]
#### Summary
Vibration notification settings do not get saved, still getting vibrations when setting turned off.

#### Environment Information

- Device Name: LG Nexus 5X, Huawei Mate 9
- OS Version: 8.1, 7.0
- Mattermost App Version: 1.8.0
- Mattermost Server Version: 4.8.1

#### Steps to reproduce
Turn off vibration in notification settings and receive any notification.

#### Expected behavior
See above. I don't think this really needs more thorough explanation

#### Observed behavior
Another user observed a change in the vibration pattern when turning the setting on and off.

**Comment by amyblais** [comment]
Hi @svelle, thank you for reporting this!

I will try the repro steps and report back here my findings.

**Comment by amyblais** [comment]
Quick update on this - I realized that this is on Android, so I will ask someone on our team with Android to test this.

**Comment by amyblais** [comment]
Hi @svelle, a team member tested this and they were unable to reproduce the issue. My recommendation is to upgrade to 4.10 or another option is to test on [our nightly build server](https://pre-release.mattermost.com/signup_user_complete/?id=f1924a8db44ff3bb41c96424cdc20676) to see if you can reproduce this there. Would one of these options sound good?

**Comment by svelle** [comment]
Okay. I'll try that later today and report back to you. I'll also see if I
can take a short video of the bug.

**Comment by svelle** [comment]
Sorry for the late reply. I got unexpectedly busy last week. I did some more checking and could confirm the same behavior with the mobile-app on a Xiamoi running Lineage 15.1 (Android 8.1).  
I also experienced this behavior using different Clients, with the same latest app version, with different Servers (4.8-4.10). I think this could also be an issue with the react-native implementation of the vibration interface not just with the vibration setting in mattermost-mobile.  
@amyblais did the colleague that tested this have a Samsung phone, by any chance? Because we tested this on a Galaxy Note 8 and that was the only phone so far that did not experience this behavior.

**Comment by amyblais** [comment]
Hi @svelle! The colleague tested on a Galaxy S6 as well as on Moto G4 S, but he is not able to repro this. 

As next steps, can you help:

1. Confirm if you are seeing the issue only on notifications while the app is in the background or does it also happen when the test notification is pushed from the settings area?

2. Double check that the affected devices do not have the ringer set to vibrate? If there is an alert sound selected (as is the default), the device would still vibrate if that's what the ringer is set to.

**Comment by svelle** [comment]
The first point I can definitely confirm. It happens when testing and also with real notifications out of the app.  
But that second point might be it, it stops vibrating as soon as I go into regular ringer mode (not vibrate). I could only check this on my Nexus right now. But will report back when I can test on other devices tomorrow. I do feel rather stupid now if that's the reason why it kept vibrating  :smile:. Even though I find that behavior a bit irritating seeing as I actively turned off the vibrations. But I guess that's something out of our hands cause it's probably an Android Notification thing.
  
I did run into another bug, though. When selecting the ringtone and selecting 'Default (None)' It plays a ringtone and continues to play that ringtone until I close the app. Closing the ringtone selector does not stop the ringing. I'll see tomorrow if this also happens on other phones and then open up another issue. 
If I can confirm this with the other phones tomorrow I guess I can close this. Unless there is a way to really make the notifications silent even in Vibrate mode because in my opinion this should be expected behavior when (de-)activating such a setting.
